https://issues.redhat.com/browse/ACM-12798 (#6782)
open-cluster-management/rhacm-docs: release_notes/whats_new.adoc

Cluster lifecycle components and features are within the {mce-short}, which is a software operator that enhances cluster fleet management. The {mce-short} supports {ocp-short} and Kubernetes cluster lifecycle management across clouds and data centers. {ocp-short} is a prerequisite for this technology.

View tasks and support information at link:../clusters/cluster_mce_overview.adoc#cluster_mce_overview[Cluster lifecycle overview].

> //cleared by MJ | 08/13

Reviewer: Should this line be removed?

Author: Keeping the hidden comment for the doc reviewer to be aware

[#observability-whats-new]
== Observability

See link:../observability/observe_environments_intro.adoc#observing-environments-intro[Observability service introduction].

[#governance-whats-new]
== Governance

> * To make the content of your resources more accessible, you can now use the `getNodesWithExactRoles` function to receive a list of nodes, and use the `hasNodesWithExactRoles` function to receive confirmation about clusters that contain nodes with only roles that you have specified. See link:../governance/template_functions.adoc##template-functions[Template functions] for more details.

Reviewer 2: I'm thinking something along these lines, but feel free to change it.
```suggestion
* To configure a cluster based on the available node roles, you can now use the `getNodesWithExactRoles` function to receive a list of nodes, and use the `hasNodesWithExactRoles` function to receive confirmation about clusters that contain nodes with only roles that you have specified. See link:../governance/template_functions.adoc##template-functions[Template functions] for more details.
```

Author: Is it important to say that it's dynamic? If not, I would just remove that word

Reviewer 2: Sure, that'd be ok.

> * To configure a cluster based on the available node roles, you can now use the `getNodesWithExactRoles` function to receive a list of nodes, and use the `hasNodesWithExactRoles` function to receive confirmation about clusters that contain nodes with only roles that you have specified. See link:../governance/template_functions.adoc#template-functions[Template functions] for more details.

Reviewer 2: ```suggestion
* To configure a cluster based on the available node roles, you can now use the `getNodesWithExactRoles` function to receive a list of nodes, and use the `hasNodesWithExactRoles` function to receive confirmation about clusters that contain nodes with only the roles that you specified. See link:../governance/template_functions.adoc#template-functions[Template functions] for more details.
```

> * To configure a cluster based on the available node roles, you can now use the `getNodesWithExactRoles` function to receive a list of nodes, and use the `hasNodesWithExactRoles` function to receive confirmation about clusters that contain nodes with only roles that you have specified. See link:../governance/template_functions.adoc##template-functions[Template functions] for more details.

Author: ```suggestion
* To configure a cluster based on the available node roles, you can now use the `getNodesWithExactRoles` function to receive a list of nodes, and use the `hasNodesWithExactRoles` function to receive confirmation about clusters that contain nodes with only roles that you have specified. See link:../governance/template_functions.adoc#template-functions[Template functions] for more details.
```
